js/main.js

```diff
@@ -103,7 +103,7 @@ requirejs.config({
 		"negative-controls": "components/negative-controls",
 		"d3": "d3.min",
 		"d3_tip": "d3.tip",
-		"jnj_chart": "jnj.chart",
+		"jnj_chart": "https://cdn.rawgit.com/odysseusinc/atlascharts/master/index",
```

Comment:
Agreed. The library should be included locally (i.e. not externally) until we can move it to a OHDSI site. Since new functionality should be added to Atlas (Data Sources) not AchillesWeb *which is in maintenance and not a focus of new features moving forward* this should be fine.